front/app/components/Form/Components/VerificationIcon.tsx

```diff
@@ -6,7 +6,7 @@ import { FormattedMessage } from 'utils/cl-intl';
 
 import messages from '../messages';
 
-export default ({ show }: { show: boolean }) => {
+const VerificationIcon = ({ show }) => {
```

Comment:
Lets re-add the prop here

```suggestion
const VerificationIcon = ({ show }: { show: boolean }) => {
```

Reply:
Thanks, missed that one.